internal/actions/pr.go

```diff
@@ -234,8 +235,25 @@ func CreatePullRequest(
 			commits = append(commits, *commit)
 		}
 
-		// We need to open an editor to ask the user. Try to populate with
-		// reasonable defaults here.
+		// If a saved pull request description exists, use that.
+		saveFile := filepath.Join(os.TempDir(), fmt.Sprintf("av-pr-%s.md", sanitize.FileName(opts.BranchName)))
```

Comment:
Can we use avDir https://github.com/aviator-co/av/blob/7d8e52eff86de083a948426f91d24b110147b479/cmd/av/stack_sync.go#L351 instead of the TempDir. tmpfs is ephemeral.

Reply:
I think tempdir makes sense here. It's decently ephemeral data and it's also possible that if they fail to create a PR using av they'll just create it on GH and we'll never clean up that old data. It's not liable to be more than a few kb for most people but still I think tempfs makes sense here.

Reply:
Ephemeral here means it won't survive after reboot. When a machine has a network issue and has to reboot, after reboot the message is lost.

Reply:
Yeah, I think that's a fine trade off to make here. Most people don't reboot very often and it's likely (at least I imagine) that if you fail to open a PR you'll deal with it before the next time you'd reboot your machine.